app/services/nytimes_service.rb

```diff
@@ -0,0 +1,19 @@
+require 'faraday'
+require 'json'
+
+class NytimesService
+  def get_obituaries
+    JSON.parse(conn.get.body, symbolize_names: true)
+  end
+
+  private
+
+  def conn
+    Faraday.new(url:'https://api.nytimes.com/svc/search/v2//articlesearch.json') do |f|
+      f.params['api-key'] = ENV['NYT_API_KEY']
```

Comment:
I wasn't able to get figaro or dotenv working, so I decided to just store the API key in my `.bash_profile` If anyone wants to run the tests for this you will need to add it to your bash profile until we find a way to securely store it in the app.

Reply:
Another option would be to just store the key in our rails app and pass it as a param to the microservice.

Reply:
Would you be able to store it in a random file and then put that file in the .gitignore file?

Reply:
I'm not sure, I was trying to put it in the `application.yml` file and that was in the gitignore, but It wasn't working. We could just put it in a ruby file and have that in the gitignore, but I'm not sure if that would work as an ENV variable or be very secure. Maybe this is something we can bring up with Dione on Monday.

Reply:
yeah, good idea

Reply:
I was helping Sage with facebook at one point and my application.yml wasn't working, and then it was in gitignore but still pushed to github so feeling like there are a lot of mysteries to how we use/hide keys.

Reply:
Which is to say putting it in your bash profile was so smart! once we deploy to Heroku this will be a non-issue, right?

Reply:
Nice work here Jack!